docs/pbc/all/ratings-reviews/202212.0/third-party-integrations/integrate-bazaarvoice.md

```diff
@@ -50,10 +51,17 @@ You can do the administration work on the Bazaarvoice reviews from the [Bazaarvo
 
 ### 2. Add Bazaarvoice domain to your allowlist
 
-To enable your customers to leave reviews on your products, you must add the Bazaarvoice domain to your **Content Security Policy** allowlist. To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
+To enable your customers to leave reviews on your products, you must add the Bazaarvoice domain to your **Content Security Policy** allowlist. 
 
-In the `deploy.yml` file, introduce the required changes: 
 
+To do that, change your `deploy.yml` file or your `config/Shared/config_default.php` file if changing the environment variable is not possible.
+
+Update `config/Shared/config_default.php` file. If you updated the `deploy.yml` file, then this step can be ignored.
+```php
+$config[KernelConstants::DOMAIN_WHITELIST][] = '*.bazaarvoice.com';
+```
+
+Or update `deploy.yml` file (which is used for your project domain):
```

Comment:
@danielsantos-spryker do I understand correctly, that to add the Bazaarvoice domain, you can do one of the following:

- either update the `config/Shared/config_default.php` file
- or update the `deploy.yml` file?

Reply:
yes